detect_meteors.py

```diff
@@ -176,11 +182,52 @@ def detect_meteors(rf_dir, id_dir, noise_dir, output_dir,
         tmm.loadFromHdf5('/tmp/tmm.hdf5', skip_lowlevel=True)
     else:
         tmm.loadFromHdf5(skip_lowlevel=True)
-
-    for k, (tx, rx) in enumerate(data_generator(rfo, ido, no, tmm, s0, s1, rxch, txch)):
-        #FIXME call processing functions here
-        pass
-
+    
+    data_list = []
+    saved_data = np.zeros((59858, 1120))
+    pulse_data = data_generator(rfo, ido, no, tmm, s0, s1, rxch, txch)
+    times = []
+    for k, (tx, rx) in enumerate(pulse_data):
+
+        num_of_pulses = np.int((s1 - s0)/(rx.sample_rate*0.0267300605774))
```

Comment:
Magic constants are bad, since I have no idea what 0.026... represents. Better to calculate it from defined variables.

Also, since this does not depend on k, tx, or rx, it should be outside the for loop so it doesn't get needlessly recalculated.